src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useCallback } from 'react';
+import classNames from 'classnames';
+import { TypeOfFiltering } from '../../types/TypeOfFiltering';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  setFilterType: (x:TypeOfFiltering) => void;
+  filterType: TypeOfFiltering;
+};
+
+export const Footer:React.FC<Props> = ({
+  todos,
+  setFilterType: setTypeOfFiltering,
+  filterType: typeOfFiltering,
+}) => {
+  const notCompletedTodosCount = todos.filter(todo => !todo.completed).length;
+
+  const isDisabled = useCallback(() => {
+    return todos.find(todo => todo.completed) === undefined;
+  }, [todos]);
```

Comment:
Seems that you don't need a function here. It should be a value. Here is my suggestion 
```suggestion
  const isCompleted = useMemo(() => {
    return !!todos.find(todo => todo.completed);
  }, [todos]);
```